docusaurus/docs/operate/Getting Started/1_quickstart.md

```diff
@@ -0,0 +1,111 @@
+---
+sidebar_position: 1
+title: Quick Start Guide (<10 minutes)
+description: Guide to get a PATH instance up and running.
+---
+
+This guide will help you set up and run PATH to serve requests using Morse or Shannon protocol in under 10 minutes.
+
+> **Note:** This guide covers running PATH without any authentication or authorization mechanisms. Work is underway to provide these capabilities - see [PATH Guard documentation](https://path.grove.city/operate/helm/guard) for details.
+
+## Prerequisites
+
+- Docker installed and running on your system
+
+## 1. Prepare Your Configuration
+
+First, prepare your configuration file by following [the PATH Config File instructions](https://path.grove.city/develop/path/configurations_path).
+
+1. Create a config directory for PATH:
+   ```bash
+   mkdir -p ./path/config
+   ```
+
+2. Copy your configuration file to the new directory with the correct name:
+   ```bash
+   export CONFIG_FILE=/PATH/TO/YOUR/CONFIG/FILE
+   cp $CONFIG_FILE ./path/config/.config.yaml
+   ```
+
+## 2. Set Up the PATH Container
+
+1. Set the PATH container image version from [the set of available tags](https://github.com/buildwithgrove/path/pkgs/container/path):
+   ```bash
+   export PATH_IMAGE_TAG='sha-9a877a0-rc'
+   ```
+
+2. Start the PATH container:
+   ```bash
+   docker run \
+     -itd \
+     --name path \
+     -p 3069:3069 \
+     -v ./path/config:/app/config \
+     ghcr.io/buildwithgrove/path:$PATH_IMAGE_TAG
+   ```
+
+   Parameter explanation:
+   - `-p 3069:3069`: Map port 3069 on host to port 3069 in container. PATH listens on port 3069 for requests.
+   - `-v ./path/config:/app/config`: Mount the config directory. PATH expects its configuration file at `/app/config/.config.yaml`
+
+## 3. Verify PATH is Running
+
+1. Check the PATH container logs:
+   ```bash
+   docker logs path
+   ```
+
+2. Wait for PATH to be ready to serve requests:
+   ```bash
+   curl -s http://localhost:3069/healthz | jq '.status'
```

Comment:
IMO the rest of the response from the `healthz` endpoint is useful for the user as well. Not super important but the full response body would indicate to users that the protocol and hydrator are being waited for which could be useful context.